rust/nasl-interpreter/src/scheduling/mod.rs

```diff
@@ -272,8 +272,8 @@ where
             .collect::<Vec<_>>();
         let mut results = core::array::from_fn(|_| E::default());
         let mut vts = Vec::new();
-        let mut unresolved_dependencies = Vec::new();
-        let mut resolved_dependencies = HashMap::new();
+        let mut unknown_dependencies = Vec::new();
+        let mut found_dependencies = HashMap::new();
```

Comment:
Nitpicking, but maybe rename this to `known_dependencies`

Reply:
fair enough :)